### PR TITLE
New Vertical Tab UX refactor

### PR DESCRIPTION
**Pull request by prakashsvmx**
Issue Description:
https://github.com/miniohq/engineering/issues/306

<summary>  How does it look ? some screens:  </summary>

<details> 
user_details

Object_details_new

policy_details

bucket_details

iam_policy_list

Mobile Preview:
image

</details>

Note:
Icons, and other listing screens will be updated in subsequent PR.

### CODE REVIEW

**Comment by dvaldivia** [comment]
after rebase, the tests should pass @prakashsvmx

**Comment by dvaldivia** [comment]
Do you think we can make the component expand to the bottom of the screen? to fill all this remaining space

Screen Shot 2021-11-10 at 4 58 06 PM

**Comment by prakashsvmx** [comment]
@dvaldivia 
the tab content area is designed with 100% height that, it will shrink/grow based on the content.
for example: ( the summary section height is larger than available view port height)

image

**Comment by prakashsvmx** [comment]
@dvaldivia @bexsoft @adfost @Alevsk @jinapurapu  - This is ready for review. PTAL

Impacted Screens:
- MinIO Console
  - Buckets -> New Bucket -> Manage
  - Buckets -> Browse -> Upload Object -> Object Details
  - Users -> User Details
  - Groups -> Group Details
  - IAM Policies -> Policy Details
- Operator Console
  -  Tenant details
  -  Storage

**Review by dvaldivia — APPROVED** [review]
tested. lgtm

File: portal-ui/src/screens/Console/Tenants/TenantDetails/VolumesSummary.tsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ const styles = (theme: Theme) =>
       color: "black",
     },
     tableWrapper: {
-      height: "calc(100vh - 267px)",
+      height: "450px",
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/VolumesSummary.tsx:55]: This had a calc to make the container full height size

File: portal-ui/src/screens/Console/Users/ListUsers.tsx [hunk 2]
```diff
@@ -82,7 +83,7 @@ const styles = (theme: Theme) =>
       },
     },
     twHeight: {
-      minHeight: 600,
+      minHeight: "450px",
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Users/ListUsers.tsx:86]:
```suggestion
      minHeight: 450,
```

File: portal-ui/src/screens/Console/Users/UserDetails.tsx [hunk 3]
```diff
@@ -49,16 +48,32 @@ import history from "../../../history";
...
+import FormSwitchWrapper from "../Common/FormComponents/FormSwitchWrapper/FormSwitchWrapper";
+import BackLink from "../../../common/BackLink";
 
 const styles = (theme: Theme) =>
   createStyles({
+    pageContainer: {
+      border: "1px solid #EAEAEA",
+    },
+    tableHeight: {
+      height: "400px",
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Users/UserDetails.tsx:66]:
```suggestion
      height: 400,
```

Please review table sizes, as I see that these values are hardcoded heights.

**Review by bexsoft — CHANGES_REQUESTED** [review]
There is an issue in collaved view, tabs container are not center aligned & there is a missing border at the bottom of the container
Screen Shot 2021-11-11 at 15 06 56 copy
:

**Review by adfost — DISMISSED** [review]
Handle merge conflicts.

**Comment by prakashsvmx** [comment]
@bexsoft  Thank you for the detailed review. I have addressed the comments. PTAL.

Quick summary:
1. https://github.com/minio/console/pull/1210#pullrequestreview-804219064 - this is due to the short text. if the text is few characters long, it looks fine.
2. table height has been removed. ( calc is not a viable solution as we have tables at different levels). so iit has been left to the defaults of the `TableWrapper`.

**Review by dvaldivia — APPROVED** [review]
Tested. LGTM.

**Review by bexsoft — APPROVED** [review]
LGTM

**Pull request merged**